pages/test-evaluate/evaluation-tools/index.pl.md

```diff
@@ -0,0 +1,136 @@
+   <p><strong>Data:</strong> Aktualizacja 28 kwietnia 2020.</p>
+   <p><strong>Redakcja:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
+  <p>Film opracowany przez Grupę Roboczą ds. Edukacji i Promocji (<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>) przy wsparciu projektu <a href="https://www.w3.org/WAI/about/projects/wai-guide/">WAI-Guide</a> finansowanego przez Komisję Europejską (KE) w ramach programu Horyzont 2020 (Umowa o dotację 822245).</p>
+---
+
+Narzędzia do oceny dostępności cyfrowej to programy lub usługi online, które pomagają określić, czy treść internetowa spełnia standardy dostępności.
+
+<div class="video-card" id="video-intro">
+  {% include video-player-data.html
+      video-id="tools-for-evaluating"
+      yt-id="bn1XJSjc_qM"
+  %}
+  <p><a href="#alternatives">Transkrypcja filmu „Omówienie narzędzi oceny” </a></p>
+</div>
+
+[Lista narzędzi oceny dostępności cyfrowej](/test-evaluate/tools/list/)
+:   Zawiera informacje na temat ponad 100 narzędzi. Możesz skorzystać z filtrów, aby zawęzić listę do typów narzędzi, które Cię interesują.
+
+[[Wybór narzędzi oceniających dostępność cyfrową]](/test-evaluate/tools/selecting/)
+:   Zawiera wskazówki dotyczące wyboru narzędzi. Opisuje cechy i funkcjonalność różnych typów narzędzi oceny oraz omawia kwestie, które należy wziąć pod uwagę w konkretnej sytuacji.
+
+Powiązane materiały przedstawiono w: [[Omówienie oceniania dostępności internetowej]](/test-evaluate/)
+
+<hr>
+
+{% include excol.html type="start" id="video-intro-transcript" %}
+
+##  Omówienie narzędzi diagnostycznych {#alternatives}
+
+{% include excol.html type="middle" %}
+
+_Ten film jest również dostępny na serwerze W3C: Film: Omówienie narzędzi oceny (format pliku: MP4, rozmiar pliku: 51MB MB)](https://media.w3.org/wai/evaluation-intros/tools-for-evaluating.mp4)._
+
+###  Transkrypcja tekstowa z opisem elementów wizualnych dla filmu Wybór i stosowanie narzędzi oceny {#transcript}
+
+Filmy zawierają podstawową animację ilustrującą to, co mówi głos spoza ekranu. Ludzie są przedstawieni za pomocą symboli graficznych.
+
+<table aria-labelledby="transcript">
+  <thead>
+    <tr>
+      <th width="50%">Dźwięk</th>
+      <th>Obraz</th>
+    </tr>
+  </thead>
+<tbody>
+  <tr>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+  </tr>
+  <tr>
+    <td>Istnieją programy i usługi internetowe, które pomogą Ci rozpoznać problemy z dostępnością.</td>
+    <td>Otwiera się skrzynka narzędziowa przy komputerze. Ze skrzynki wychodzi lupa, aby sprawdzić stronę internetową na komputerze. Lupa powiększa słowo „dostępność”.</td>
```

Comment:
@zwiastunsw The description seems to have been reworded. Similar to https://github.com/w3c/wai-website/pull/561#discussion_r1679352606